Standards/scs-XXXX-vN-decision-record-template.md

```diff
@@ -0,0 +1,40 @@
+---
+title: _Descriptive title_
+type: Decision Record
+status: Draft
+track: Global  # | IaaS | Ops | KaaS | IAM
+author: Author <Mail or Github handle>
+date: DD-MM-YYYY
+---
+
+<!---
+This is a template striving to provide a starting point for
+creating a standard or decision record adhering to scs-0001.
+Replace at least all text which is _italic_.
+See https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0001-v1-sovereign-cloud-standards.md
+--->
+
+## Introduction
+
+Introduction
+
+## Terminology
+
+Terminology
```

Comment:
This needs - imho - more explanation or an example.

Reply:
I had something in mind like its done in some RFC documents e.g. [HTTP](https://www.rfc-editor.org/rfc/rfc2616) with their section `Terminology`
But how exactly this section is built up is up for discussion. I would like to do it like the mentioned `RFC` section, but I'm also on board with the previously used table structure.

Reply:
Updated the section to contain an example for a terminology.